.github/CONTRIBUTING.md

```diff
@@ -7,6 +7,15 @@
 
 ## Testing
 
+### Setup
```

Comment:
Yeah, though it only really needs to be done once. If you're iterating with `npm test` it would  slow things down. The linking part can go away once the packages have been published to NPM, at which point this script would just be running `npm install` in a bunch of folders.